tests/interchain/provider_test.go

```diff
@@ -101,6 +101,52 @@ func (s *ProviderSuite) TestProviderCreateConsumerRejection() {
 	s.Require().Error(err)
 }
 
+// TestOptInChainCanOnlyStartIfActiveValidatorOptedIn tests that only if an active validator opts in to an Opt-In chain, the chain can launch.
+// Scenario 1: Inactive validators opts in, the chain does not launch.
+// Scenario 2: Active validator opts in, the chain launches.
+func (s *ProviderSuite) TestOptInChainCanOnlyStartIfActiveValidatorOptedIn() {
+	testAcc := s.Provider.TestWallets[2].FormattedAddress()
```

Comment:
I'll create a separate PR so that we don't have to be aware of which index is in use during the tests, and it will be handled under the hood. For now, I would use an account that isn't used in other tests, for example, account s.Provider.TestWallets[12] (just to avoid an error where an invalid account number is sent). I'll try to fix this as soon as possible since the current solution isn't the most elegant.